deployment/CP04OSSM_Deploy.csv

```diff
@@ -86,86 +86,86 @@ AT-31,,AR-04,,CP04OSSM-RID27-03-CTDBPC000,3,1,2015-10-22T19:38:00,2016-05-13T17:
+AR-04,,AR-08,,CP04OSSM-RIC21-00-CPMENG000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGCON-NCPM02-00011,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID26-00-DCLENG000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGCON-NDCL26-00015,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID26-04-VELPTA000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-VELPTA-11412,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID26-06-PHSEND000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-PHSEND-P0096,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID26-07-NUTNRB000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-NUTNRB-00266,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID26-08-SPKIRB000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-SPKIRB-00271,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID27-00-DCLENG000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGCON-NDCL27-00011,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID27-01-OPTAAD000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-OPTAAD-00189,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID27-02-FLORTD000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-FLORTD-01222,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID27-03-CTDBPC000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-CTDBPC-50003,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-RID27-04-DOSTAD000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-DOSTAD-00517,39.94003,-70.87743,,7,446,
+AR-04,,AR-08,,CP04OSSM-SBC11-00-CPMENG000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGCON-BCPM01-50001,39.94003,-70.87743,,0,446,
+AR-04,,AR-08,,CP04OSSM-SBD11-00-DCLENG000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGCON-BDCL11-50001,39.94003,-70.87743,,0,446,
+AR-04,,AR-08,,CP04OSSM-SBD11-01-MOPAK0000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-MOPAK0-44005,39.94003,-70.87743,,0,446,
+AR-04,,AR-08,,CP04OSSM-SBD11-02-HYDGN0000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-HYDGN0-99999,39.94003,-70.87743,,0,446,
+AR-04,,AR-08,,CP04OSSM-SBD11-06-METBKA000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-METLGR-00015,39.94003,-70.87743,,0,446,
+AR-04,,AR-08,,CP04OSSM-SBD12-00-DCLENG000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGCON-BDCL12-50001,39.94003,-70.87743,,0,446,
+AR-04,,AR-08,,CP04OSSM-SBD12-03-HYDGN0000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-HYDGN0-99999,39.94003,-70.87743,,0,446,
+AR-04,,AR-08,,CP04OSSM-SBD12-04-PCO2AA000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-PCO2AA-34222,39.94003,-70.87743,,0,446,
```

Comment:
Reviewed corrections to recovery CUID, sensor UIDs and deployment depths. All look good.